assets/materials/shaders/PBR.hlsl

```diff
@@ -32,42 +28,39 @@ Texture2D    IBLTex         : register(IBL_MAP_TEXTURE_REGISTER,    MATERIAL_RES
 Texture2D    EnvMapTex      : register(ENV_MAP_TEXTURE_REGISTER,    MATERIAL_RESOURCES_SPACE);
 SamplerState ClampedSampler : register(CLAMPED_SAMPLER_REGISTER,    MATERIAL_RESOURCES_SPACE);
 
-//
-// GGX/Towbridge-Reitz normal distribution function with
-// Disney's reparametrization of alpha = roughness^2
-//
-float DistributionGGX(float cosLh, float roughness)
+float Distribution_GGX(float3 N, float3 H, float roughness)
 {
-    float alpha   = roughness * roughness;
-    float alphaSq = alpha * alpha;
-    float denom   = (cosLh * cosLh) * (alphaSq - 1.0) + 1.0;
-    return alphaSq / (PI * denom * denom);
+    float NoH    = saturate(dot(N, H));
+    float NoH2   = NoH * NoH;
+    float alpha2 = max(roughness * roughness, EPSILON);
+    float A      = NoH2 * (alpha2 - 1) + 1;
+	return alpha2 / (PI * A * A);
```

Comment:
Is this supposed to be indented with extra space? Here and in the next function below (Geometry_SchlickBeckman)

Reply:
The HLSL code doesn't get fed through an automatic formatter. clang-format doesn't play nice with HLSL yet. So it's just subjected to whatever formatting it came from. This code was copied over from some of my other projects.

Reply:
Also, we don't currently have an HLSL style guide :(

Reply:
Filed https://github.com/google/bigwheels/issues/108